src/templater.rs

```diff
@@ -443,3 +444,14 @@ impl TemplateProperty<Signature, Timestamp> for SignatureTimestamp {
         context.timestamp.clone()
     }
 }
+
+pub struct EmptyProperty<'a> {
+    pub repo: RepoRef<'a>,
+}
+
+impl TemplateProperty<Commit, bool> for EmptyProperty<'_> {
+    fn extract(&self, context: &Commit) -> bool {
+        context.parents().len() == 1
```

Comment:
Could use comments and/or intermediate variables here for readability. Looks like the first one is to ensure no merge and the 2nd is checking that the content hash is unchanged?

Reply:
That's right. I'll add a comment.